Tests/Unit/NetworkServer/LnsProtocolMessageProcessorTests.cs

```diff
@@ -297,7 +297,7 @@ public async Task InternalHandleDataAsync_ShouldProperlyCreateLoraPayloadForUpdf
                                                 'MIC':45234788,'RefTime':0.000000,'DR':5,'Freq':868300000,'upinfo':{'rctx':0,'xtime':68116944405337035,
                                                 'gpstime':0,'fts':-1,'rssi':-53,'snr':8.25,'rxtime':1636131701.731686}}");
             var expectedRadioMetadata = GetExpectedRadioMetadata();
-            var expectedMhdr = new byte[] { 128 };
+            var expectedMhdr = new MacHeader(128);
```

Comment:
I think we should use the more meaningful constructor here:

```suggestion
            var expectedMhdr = new MacHeader(MacMessageType.ConfirmedDataUp);
```